templates/marketplace-suggestions/container.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
This template should not be in the frontend templates directory. We do not want to support theme overrides of this file.

Reply:
(now moved)